docs/how-to/openstack-runner.md

```diff
@@ -8,6 +8,19 @@ enabled the charm cannot be changed to use other virtualization methods.
 
 There are three configuration that the charm needs to be deployed with to enable OpenStack integration: `openstack-clouds-yaml`, `openstack-flavor`, and `openstack-network`.
 
+## Integration
+
+The OpenStack runners require image to be available via the `image` integration. Deploy the
```

Comment:
Nitpick: Add "the", as in, "The OpenStack runners require *the* image to be available..."